Modules/socketmodule.c

```diff
@@ -6179,11 +6172,13 @@ socket_dup(PyObject *self, PyObject *fdobj)
     if (newfd == INVALID_SOCKET)
         return set_error();
 
+#ifndef MS_WINDOWS_NON_DESKTOP
```

Comment:
Instead of calling `SetHandleInformation()`, the above `WSASocketW()` call should simply use the flag `WSA_FLAG_NO_HANDLE_INHERIT`.

Reply:
Sure. I noticed that other calls release the gil before calling `WSASocketW`. Is there a reason this is not done for this call?